retry/src/main/java/io/micronaut/retry/intercept/DefaultRetryInterceptor.java

```diff
@@ -244,11 +244,11 @@ private Object retrySync(MethodInvocationContext<Object, Object> context, Mutabl
                             try {
                                 eventPublisher.publishEvent(new RetryEvent(context, retryState, e));
                             } catch (Exception e1) {
-                                LOG.error("Error occurred publishing RetryEvent: " + e1.getMessage(), e1);
+                                LOG.error("Error occurred publishing RetryEvent: {}", e1.getMessage(), e1);
                             }
                         }
                         if (LOG.isDebugEnabled()) {
-                            LOG.debug("Retrying execution for method [{}] after delay of {}ms for exception: {}", context, delayMillis, e.getMessage());
+                            LOG.debug("Retrying execution for method [{}] after delay of {}ms for exception: {}", context, delayMillis, e.getMessage(), e);
```

Comment:
In retry stacktraces add noise.
```suggestion
                            LOG.debug("Retrying execution for method [{}] after delay of {}ms for exception: {}", context, delayMillis, e.getMessage());
```